src/vision/frc_vision_final.py

```diff
@@ -99,49 +111,52 @@ def unequal(new, old_list):
     # Find lines in selected image
     LINES = cv2.HoughLinesP(MASK_EDGES, 1, radians(.5), 25, maxLineGap=25)
 
+    # If there are lines:
     if LINES is not None:
         NUM_LINES = len(LINES)
         FILTERED_LINES = []
         X_TOTAL = 0
         Y_TOTAL = 0
+
         for NEW_LINE in LINES:
             x1, y1, x2, y2 = NEW_LINE[0]
             new_slope = degrees(np.arctan((y2 - y1)/(x2 - x1)))
+
+            # Checks if we have verified lines, and makes a new line based on that.
             if FILTERED_LINES:
-                if (new_slope < -40 or new_slope > 40) and unequal(new_slope, FILTERED_LINES):
-                    FILTERED_LINES.append(NEW_LINE)
-                    cv2.line(FILTERED_LINE_IMG, (x1, y1),
-                             (x2, y2), (0, 255, 0), 1)
-                    X_TOTAL += x1 + x2
-                    Y_TOTAL += y1 + y2
+                if (new_slope < -angleThreshold or new_slope > angleThreshold) and unequal(new_slope, FILTERED_LINES):
+                    X_TOTAL, Y_TOTAL = newLine(
+                        FILTERED_LINES, NEW_LINE, FILTERED_LINE_IMG, x1, y1, x2, y2)
             else:
-                if new_slope < -40 or new_slope > 40:
-                    FILTERED_LINES.append(NEW_LINE)
-                    cv2.line(FILTERED_LINE_IMG, (x1, y1),
-                             (x2, y2), (0, 255, 0), 1)
-                    X_TOTAL += x1 + x2
-                    Y_TOTAL += y1 + y2
+                if new_slope < -angleThreshold or new_slope > angleThreshold:
+                    newLine(FILTERED_LINES, NEW_LINE,
+                            FILTERED_LINE_IMG, x1, y1, x2, y2)
 
         NUM_LINES = len(FILTERED_LINES)
         if FILTERED_LINES:
             X_AVG = 0
             Y_AVG = 0
+
             if len(X_VALS) == POINT_SAMPLES:
                 X_VALS[pointer] = X_TOTAL/(2*NUM_LINES)
                 Y_VALS[pointer] = Y_TOTAL/(2*NUM_LINES)
+
                 for i in range(len(X_VALS)):
                     X_AVG += X_VALS[i]
                     Y_AVG += Y_VALS[i]
+
                 X_AVG = int(X_AVG / POINT_SAMPLES)
                 Y_AVG = int(Y_AVG / POINT_SAMPLES)
-                sd.putBoolean("seesTarget", True)
 
                 offset = 2 * (X_AVG - (WIDTH/2)) / WIDTH
+                cv2.circle(FILTERED_LINE_IMG, (X_AVG, Y_AVG),
+                           5, [255, 255, 255], -1)
+                dist_to_target = depth.get_distance(X_AVG, Y_AVG)
+
+                # Smart Dashboard variables
+                sd.putBoolean("seesTarget", True)
```

Comment:
Just another naming consistency issue, this should either be `Sees Target` or everything else should be changed to camelCase